tests/runtime/flavorful/host.ts

```diff
@@ -75,4 +71,5 @@ async function run() {
   assert.deepStrictEqual(r2, ['typedef4']);
 }
 
-await run()
+// TLA cycle avoidance
```

Comment:
What is "TLA"? (curious, haven't seen this before)

Reply:
Top-level await. Sorry forget it's not widely known!